logistics/scripts/modules/database/database-management.psm1

```diff
@@ -82,7 +82,11 @@ function New-DbConnectionStringBuilder {
         $newDbCSB = New-DbConnectionStringBuilder
         foreach ($key in $existingCSB.keys) {
             $value = $existingCSB[$key]
-            if (-not ($emptySpecificCSB[$key] -eq $value)) {
+            if($key -eq 'Encrypt')
```

Comment:
Could you give some details on why these PowerShell changes were needed?

Reply:
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/pull/645

Reply:
It related with Connection string issue

Reply:
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/actions/runs/7649445186/job/20843860141#step:12:65

Invoke-SqlScript: ERROR: Microsoft.SqlServer.Management.Common.ConnectionFailureException: Failed to connect to server (local).  --->
Microsoft.Data.SqlClient.SqlException (0x80131904): A connection was successfully established with the server, but then
an error occurred during the login process. (provider: SSL Provider, error: 0 - The certificate chain was issued by an
authority that is not trusted.)  ---> System.ComponentModel.Win32Exception (0x80090325): The certificate chain was
issued by an authority that is not trusted.    at Microsoft.Data.SqlClient.SqlInternalConnection.OnError(SqlException
exception, Boolean breakConnection, Action`1 wrapCloseInAction)    at
Microsoft.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean